src/components/ebay-calendar/index.marko

```diff
@@ -104,10 +103,12 @@ $ const rangeEndText = input.a11yRangeEndText ?? "end of range"
                                             </button>
                                         </if>
                                         <else>
-                                            <span
-                                                class=disabled && "calendar__cell--disabled"
-                                                aria-label=rangeAriaLabel>
-                                                ${day}
+                                            $ if (isToday) a11yText += ` - ${todayText}`;
```

Comment:
How is this going to be localized? We also have to consider that in some other languages the day goes before the action. 
I think we should allow this to be fully changeable. We can give them the tools they need to do so however.

Reply:
I was thinking about this when I wrote the code, which is why I have hyphens between each label. So the longest label will be something like
```
10 - [selected] - [start of range] - [today]
```
where all of the text in `[]` brackets  is configurable. Is this not enough for localization?

Reply:
Yes to what Luke says. The idea with the hyphens is that this is not a grammatical construct/sentence. Instead we are mimicking ARIA states whereby we control the order in which they are read.